screens/AccountScreen.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import { Button, Appbar } from "react-native-paper";
+import AccountModal from "../components/AccountModal";
+
+const AccountScreen = (props) => {
+  //determines what component the modal will render when a button is clicked
+  const [state, setState] = React.useState("");
```

Comment:
This should be a more specific variable name than "state", this is too generic and confused with the general concept of state, useState(), etc. Something like "action" or "element" would be good here.